Okay, the compile error with ratecontrol.cpp is fixed. The build is failing for an unrelated reason which has been affecting lots of AppVeyor builds seemingly randomly:
LINK : fatal error LNK1181: cannot open input file 'vamp-sdk.lib'

--- a/src/preferences/dialog/dlgprefdeck.cpp
+++ b/src/preferences/dialog/dlgprefdeck.cpp
@@ -456,7 +456,7 @@ void DlgPrefDeck::slotSetTrackTimeDisplay(QAbstractButton* b) {
 }
 
 void DlgPrefDeck::slotSetTrackTimeDisplay(double v) {
-    m_timeDisplayMode = static_cast<TrackTime::DisplayMode>(v);
+    m_timeDisplayMode = static_cast<TrackTime::DisplayMode>(static_cast<int>(v));
     m_pConfig->set(ConfigKey("[Controls]","PositionDisplay"), ConfigValue(v));
     if (m_timeDisplayMode == TrackTime::DisplayMode::Remaining) {
         radioButtonRemaining->setChecked(true);

